tests/function/test_service.py

```diff
@@ -136,24 +134,19 @@ def test_virtwho_service_control_by_ssh_connect(self, virtwho, ssh_guest, ssh_ho
         server = config.virtwho.server
         port = config.virtwho.port
         ssh_access_no_password(ssh_guest, ssh_host, server, port)
+        virtwho.operate_service(action="stop")
```

Comment:
firstly stop the virt-who in virt-who host, and then start and stop it in a remote rhel host by ssh, otherwise the virt-who service often run in `failed` status.

Reply:
I am considering whether we should also add this to the `rhel-8` branch

Reply:
Yes, the case may also failed in rhel-8, but let's maintain that branch after completed all the tests of rhel9.5 and 10.0.beta, we can trigger the rhel-8 once and make code update together.